src/lavinmq/amqp/queue/message_store.cr

```diff
@@ -347,6 +347,9 @@ module LavinMQ
                 @segments.delete seg
                 next
               end
+            rescue ex
+              @log.error { "Closing message store: invalid SchemaVersion in #{path}" }
```

Comment:
how do we know that it is a "invalid SchemaVersion" error if we catch all errors? aren't we raising a specific exception then? Catch only that then, or log `ex.message`.